### ISSUE TITLE
small translation issue in french ios app

### ISSUE THREAD

**Issue by Ospeleo** [subsurface/subsurface#3236]
Hi,

in iOS app, using french translation
touching the hamburger menu (bottom left)
there is the switch option "Désactiver la synchro auto avec le cloud" ( when the synchronization is manual)
and "Activer la synchro automatic" (when synchronization is automatic)

due to the conjugation of the verb, the phrasing in french means the contrary of what it actially does.

the correct phrases should be:
"Synchro manuelle avec le cloud" (or " Synchro auto désactivée avec le cloud")
and "Synchro auto activée" or "Synchro auto activée avec le cloud "

and maybe, because the phrasing is very similar, "Synchro manuelle avec le cloud" (the icon above) could be "Synchroniser maintenant avec le cloud"
![IMG_2983](https://user-images.githubusercontent.com/12714994/116699000-0e4ac580-a993-11eb-8044-f194c81101ea.PNG)
![IMG_2984](https://user-images.githubusercontent.com/12714994/116699004-10ad1f80-a993-11eb-9834-1b0036fa3f17.PNG)

**Comment by schtibal** [comment]
I don't believe that it's an incorrect language translation - I believe that the French translation implies the same as what the English does, it's just that both the text and the icon show what will happen when you select it, not what it currently is.

Ie,
When cloud sync is currently disabled, the cloud with a tick is displayed:
![image](https://user-images.githubusercontent.com/16427056/120107423-4fa5df00-c1a4-11eb-9f86-9620f82626d7.png)
And the text is:
English: Disable auto cloud sync
French: Désactiver la synchro auto ...

When cloud sync is currently enabled, the cloud with a cross through it is displayed:
![image](https://user-images.githubusercontent.com/16427056/120107528-af03ef00-c1a4-11eb-8b0e-6fb7506c2ff7.png)
And the text is:
English: Enable auto cloud sync
French: Activer la synchro ...

Some could argue that it is counter intuitive that the icon shows a broken cloud when auto sync is disabled, and a tick when it is enabled.
Others could argue that pressing the broken cloud disables auto sync, and pressing the cloud with a tick enables it...
